.github/workflows/devnet_main_deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Main CD for Devnet Deployment
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+  push:
+    paths:
+      - "aws/devnet/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    uses: ./.github/workflows/terraform_template_deploy.yml
```

Comment:
Why `./.github` and not just `.github`?

Reply:
makes sense will fix

Reply:
It will complain in IDE that it can't find the re-usable workflow